### PR TITLE
feat: migrate preferences to new sidebar navigation

### PR DESCRIPTION
**Pull request by thisisamir98**
### Overview
With the latest enhancements in our sidebar design and implementation, this PR introduces a significant improvement by migrating the preferences functionality to seamlessly integrate with the new sidebar navigation. Previously, users accessed preferences via a settings icon, which redirected them to a separate preferences page. Now, a dedicated sidebar button for preferences has been introduced, providing users with direct access to their preferences without disrupting their navigation flow.

### Changes Made
- **Sidebar Integration:** The preferences functionality is now seamlessly integrated into the sidebar navigation.
- **Dedicated Button:** A dedicated button for preferences has been added to the sidebar, allowing users to access preferences directly.
- **Improved User Experience:** The previous fade-in-out animation has been removed, as preferences no longer take over the entire application screen upon opening. Instead, preferences now open within the sidebar, maintaining continuity with other main sections.

### Benefits
- **Enhanced Accessibility:** Users can now access preferences more conveniently through the dedicated sidebar button.
- **Streamlined Navigation:** The integration of preferences into the sidebar enhances the overall user navigation experience.
- **Smoother Transition:** Removal of the fade-in-out animation ensures a smoother transition between sections, improving user interaction.

### Additional Information
This migration aligns with our ongoing efforts to optimize user experience and streamline navigation within the application. By integrating preferences into the new sidebar design, we aim to provide users with a more intuitive and efficient way to manage their settings while maintaining a cohesive user interface.

## Screenshots/Screencast (for UI changes)
image

### CODE REVIEW

File: src/script/components/Conversation/Conversation.tsx [hunk 1]
```diff
@@ -488,7 +488,6 @@ export const Conversation = ({
             isRightSidebarOpen={isRightSidebarOpen}
             isReadOnlyConversation={showReadOnlyConversationMessage}
           />
-
```
Comment (przemvs) [inline src/script/components/Conversation/Conversation.tsx]: This changes are necessary? :)

File: src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx [hunk 2]
```diff
@@ -219,63 +213,73 @@ const Conversations: React.FC<ConversationsProps> = ({
...
-      <button
-        type="button"
-        className={cx(`conversations-settings-button accent-text`, {'conversations-settings--badge': showBadge})}
-        title={t('tooltipConversationsPreferences')}
-        onClick={onClickPreferences}
-        data-uie-name="go-preferences"
-      >
-        <Icon.Settings />
-      </button>
+  function getHeader() {
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx:216]: Maybe we can move this getHeader to another component? Conversations/ConversationsHeader or smth like that? 🤔 We will have more clear structure :)

Comment (przemvs) [reply src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx:216]: And also I think in my task I could remove user details from there :)

File: src/script/components/Conversation/Conversation.tsx (hunk 1 above)
Comment (thisisamir98) [reply src/script/components/Conversation/Conversation.tsx]: https://github.com/wireapp/wire-webapp/pull/17016/commits/2797b2776dfb55880dc27f3326014494cb10eebc

File: src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx (hunk 2 above)
Comment (thisisamir98) [reply src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx:216]: let's do this in your PR for legal hold migration

**Review by przemvs — APPROVED** [review]

**Pull request merged**